tests/test_wps_convolution.py

```diff
@@ -3,40 +3,36 @@
 from pywps.tests import client_for, assert_response_success
 from pkg_resources import resource_filename
 
-from .common import run_wps_process
+from wps_tools.testing import run_wps_process
 from osprey.processes.wps_convolution import Convolution
 
-# ********************************************************************************
-#   The test is disabled due to failues caused by RVIC's internal issues:
-#   https://github.com/UW-Hydro/RVIC/issues/96
-#   https://github.com/UW-Hydro/RVIC/issues/130
-# ********************************************************************************
-# @mark.slow
-# @mark.online
-# @mark.parametrize(
-#     ("config"),
-#     [
-#         f"{resource_filename(__name__, 'configs/convolve_opendap.cfg')}",
-#         {
-#             "OPTIONS": {
-#                 "CASEID": "sample",
-#                 "RUN_STARTDATE": "2012-12-01-00",
-#                 "STOP_DATE": "2012-12-31",
-#                 "CALENDAR": "standard",
-#             },
-#             "DOMAIN": {
-#                 "FILE_NAME": "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample_routing_domain.nc"
-#             },
-#             "PARAM_FILE": {
-#                 "FILE_NAME": "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample.rvic.prm.COLUMBIA.20180516.nc"
-#             },
-#             "INPUT_FORCINGS": {
-#                 "DATL_PATH": "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/",
-#                 "DATL_FILE": "columbia_vicset2.nc",
-#             },
-#         },
-#     ],
-# )
-# def test_wps_convolution(config):
-#     params = ("config={0};").format(config)
-#     run_wps_process(Convolution(), params)
+
+@mark.slow
+@mark.online
+@mark.parametrize(
+    ("config"),
+    [
+        f"{resource_filename(__name__, 'configs/convolve_opendap.cfg')}",
+        {
+            "OPTIONS": {
+                "CASEID": "sample",
+                "RUN_STARTDATE": "2012-12-01-00",
+                "STOP_DATE": "2012-12-31",
+                "CALENDAR": "standard",
+            },
+            "DOMAIN": {
+                "FILE_NAME": "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample_routing_domain.nc"
+            },
+            "PARAM_FILE": {
+                "FILE_NAME": "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample.rvic.prm.COLUMBIA.20180516.nc"
+            },
+            "INPUT_FORCINGS": {
+                "DATL_PATH": "https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/",
+                "DATL_FILE": "columbia_vicset2.nc",
+            },
+        },
+    ],
+)
+def test_wps_convolution(config):
+    params = ("config={0};").format(config)
+    run_wps_process(Convolution(), params)
```

Comment:
I am sorry that I have not checked the osprey repo very often. Did we decide not to comment on the tests that fail due to RVIC's issues?

Reply:
It's all good. I removed the comments because we already have `pytest` commented out on the `python-ci.yml` workflow, so I thought it'd be redundant. Though now that you mention it, we might want to at least comment the [links to the issues](https://github.com/pacificclimate/osprey/blob/3f9791d8667ad36b10785ac06be1b943b564d5eb/tests/test_wps_convolution.py#L11) and the fixes needed to get the tests working. @nikola-rados?

Reply:
I agree with you. We should add the additional instructions to the Readme.